src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -0,0 +1,205 @@
+import { useState } from 'react';
+import { v4 as uuid } from 'uuid';
+
+import { Button } from '../../common/Button/Button';
+import { getDuration } from '../../helpers/pipeDuration';
+import Input from '../../common/Input/Input';
```

Comment:
Remember about logical import order, in your case components, then helpers

Reply:
Done!